.github/workflows/build.yaml

```diff
@@ -40,6 +40,10 @@ jobs:
         with:
           dependency-versions: "${{ matrix.dependencies }}"
 
+      - name: "Check code style"
+        if: ${{ matrix.dependencies == 'highest' && matrix.php == '8.1' }}
```

Comment:
This should essentially just run on `locked` dependencies (same for psalm), but we don't run CI with locked deps, maybe we should?